Cython/Compiler/ModuleNode.py

```diff
@@ -2336,6 +2432,144 @@ def generate_import_star(self, env, code):
         code.putln(UtilityCode.load_as_string("ImportStar", "ImportExport.c")[1])
         code.exit_cfunc_scope()  # done with labels
 
+    def generate_module_state_start(self, env, code):
+        code.putln("#if CYTHON_COMPILING_IN_LIMITED_API")
+        code.putln('typedef struct {')
+        code.putln('PyObject *%s;' % Naming.builtins_cname)
```

Comment:
This should later be refactored to bring the L'API struct declarations closer to the non-L'API C global declarations again. Keeping both apart is annoying from a maintenance perspective.

Reply:
Agreed, I've added a TODO here, I should be able to get to this on later changes